CMakeLists.txt

```diff
@@ -24,7 +24,7 @@
 #   IN THE SOFTWARE.
 #
 
-cmake_minimum_required (VERSION 2.8.12)
+cmake_minimum_required (VERSION 3.5)
```

Comment:
This will break anyone still building on older platforms where CMake is < 3.5.

There is a new range syntax that is apparently supported ... cmake_minimum_required(VERSION 2.8.12 .. 3.5) or something.

Please do the research to make sure we don't break 2.8.12 compatibility as this is a legacy project which some people may still be using on older projects.